IntegrationAdaptersTests/RegisterHospitalTests.cs

```diff
@@ -0,0 +1,45 @@
+using System;
+using System.Configuration;
+using IntegrationAdapters;
+using IntegrationAdapters.Models;
+using IntegrationAdapters.Services;
+using Microsoft.EntityFrameworkCore;
+using Xunit;
+
+namespace IntegrationAdaptersTests
+{
+    public class RegisterHospitalTests
+    {
+        public readonly HealthCareSystemDbContext dbContext;
+        [Fact]
+        public void Is_pharmacy_already_existing_on_hospital()
+        {
+            var options = new DbContextOptionsBuilder<HealthCareSystemDbContext>()
+                .UseMySql(connectionString: "server=localhost;port=3306;database=newmydb;user=root;password=root")
+                .Options;
+
+            ApiService service = new ApiService(new HealthCareSystemDbContext(options));
```

Comment:
I think the convention is to call the object we are testing 'subject' which is short for 'subject under test' so that we emphasize what we are testing. Also options object can be extracted as a field because it's used in both tests - this will make the tests shorter and easier to maintain.

Reply:
Maybe you should think about using mocks in these tests instead of the real data base - that way the Apis you are using aren't dependent on the actual data in the database? Also you should add tests for pharmacy registration

Reply:
Okay, on daily assistant told us it's okay to use real databese because it is integration tests, but we need to add unit tests and correct these mistakes. Thanks